src/main/java/com/google/maps/model/AddressType.java

```diff
@@ -234,6 +234,10 @@ public String toString() {
     return addressType;
   }
 
+  public String toCanonicalLiteral() {
```

Comment:
I'd like to. I think it's more explicit that this literal resembles the literal returned by the server. `toString` doesn't seem to promise that. And `toUrlValue` seems to indicate that this can be used in a URL query string, but not that it's the literal returned by the server.